### DISCUSSION TITLE
Fundamentos do Blazor... / ... na prática / Páginas e Rotas

### DISCUSSION THREAD

**Discussion by DrenokBR** [balta-io/forum#501] [Tira Dúvidas] [answered]
Bom Dia, Balta !

Estou usando a versão 8.0.2 do DotNet e na aula intitulada "Páginas e Rotas" do curso 
"Fundamentos do Blazor com .NET 8" percebi que, mesmo usando exatamente o mesmo comando que você usou para criar o Projeto "BlazorShop", o meu template Blazor cria uma estrutura diferente da que foi criada em seu ambiente de desenvolvimento mostrado no curso. Vou deixar uma imagem comparando as duas estruturas. Minha pergunta é a seguinte: Será que isso vai atrapalhar muito o meu acompanhamento do curso? Será que seria melhor desinstalar o DotNet 8.0.2 e tentar voltar para a versão 8.0.1 ou 8.0.0 ?

![EstruturaDiferenteCursoMeuAmbiente](https://github.com/balta-io/forum/assets/36772075/41fe379a-14be-4158-806b-1bfd18fa7d15)

**Comment by DrenokBR** [comment, marked as answer]
![EstruturaDiferenteCursoMeuAmbiente2](https://github.com/balta-io/forum/assets/36772075/9ed4eeb0-fe88-4669-8800-c2a066850959)

**Reply by andrebaltieri** [reply]
@DrenokBR tranquilo, acho que não vai influenciar em nada não.. na verdade acho que eu movi "sem querer" a pasta Identity nas aulas!

Segue ai, se der algum problema só chamar aqui

**Reply by DrenokBR** [reply]
Valeu Balta !!! "Brigadão".
